images/recycler/Dockerfile.rhel

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/4.11:cli
+FROM registry.ci.openshift.org/ocp/4.16:cli
```

Comment:
This dockerfile is not used in production builds

Reply:
Yeah, I bet some should be cleaned up